Mods/ArchipelagoPlayer.SC2Mod/enUS.SC2Data/LocalizedData/GameStrings.txt

```diff
@@ -1036,6 +1046,7 @@ Button/Name/AP_HaveBarrier=Improved Barrier
 Button/Name/AP_HaveGraviticBoosters=Gravitic Boosters
 Button/Name/AP_HaveNovaGadgetDefensiveDroneSummon=[PH] Place Defensive Drone
 Button/Name/AP_Heal=Heal
+Button/Name/AP_HealingAdaption=Healing Adaption

```

Comment:
Typo here -- "adaption" should be spelled "adaptation".

The typo is in most of the data IDs as well, but that's not user facing so I'm not sure it's worth cleaning up. It's a small cost to off-the-cuff searchability but I think we can live with it.

Reply:
I messed it up everywhere, I'm just going to fix it, I'd rather not live with teh shame of seeing that in the future